pages/Types.html

```diff
@@ -604,6 +605,17 @@ <h2 id="Element"> Element </h2>
 </code></pre>
 <p>You could replace <code>this.value</code> with <code>$(this).val()</code> to access the value of the text input via jQuery, but in that case you wouldn't gain anything.
 </p>
+<h2 id="Text"> Text </h2>
+<p>Text is a node of the Document Object Model (DOM) that represents the textual content of an <a href="#Element">element</a> or an attribute. Consider the following code:
+</p>
+<pre><code data-lang="html">&lt;p id=&quot;target&quot;&gt;&lt;b&gt;Hello&lt;/b&gt; world&lt;/p&gt;
+</code></pre>
+<p>If you retrieve the children of the paragraph of the example as follows:
+</p>
+<pre><code data-lang="javascript">var children = document.getElementById( "target" ).childNodes;
+</code></pre>
+you obtain two children. The first one is the <a href="#Element">element</a> represeting the <code>b</code> tag. The second children is a text node containing the string " world".
```

Comment:
typo: represeting -> representing
also: the second **children** should be the second **child**